hack/docker/setup.sh

```diff
@@ -14,7 +14,7 @@ source "$REPO_ROOT/hack/libbuild/common/public_image.sh"
 
 APPSCODE_ENV=${APPSCODE_ENV:-dev}
 IMG=stash
-RESTIC_VER=${RESTIC_VER:-0.8.3}
+RESTIC_VER=${RESTIC_VER:-0.9.4}
```

Comment:
This can't be done. This breaks existing users.

Reply:
This has been handled in another PR